lib/board.rb

```diff
@@ -39,14 +39,26 @@ def winning_symbol
     if check_for_winning_row_of(:X, all_winning_combinations) 
       return :X 
     end
-  
+
     if check_for_winning_row_of(:O, all_winning_combinations) 
       return :O
     end
-   
+
     :unset
   end
 
+  def grid_for_display
+    grid_formation = ""
+    (0...grid.size).each do |i|
+      if grid[i].nil?
```

Comment:
Hmm, might be better to push this into whatever grid contains, could use null object pattern to remove the nil check.